pkg/query/config.go

```diff
@@ -39,3 +43,25 @@ func LoadConfigs(confYAML []byte) ([]Config, error) {
 	}
 	return queryCfg, nil
 }
+
+// BuildQueryConfig initializes and returns an Query client configuration from a static address.
+func BuildQueryConfig(queryAddrs []string) ([]Config, error) {
+	configs := make([]Config, 0, len(queryAddrs))
+	for _, addr := range queryAddrs {
+		if addr == "" {
+			return nil, errors.New("static querier address cannot be empty")
+		}
+		u, err := url.Parse(fmt.Sprintf("http://%s", addr))
```

Comment:
hmmm what if someone will want HTTPS?

Reply:
or essentially what if someone will pass with `http://query.com?`

Reply:
I have implemented it in a way how it worked in v0.10.
But maybe you are right, that it should also support passing scheme. I will try to think about something and will get back to you when it's ready.

Reply:
Yes, I am not saying what was there was the best (: That's why we always look for improvement (:

Reply:
I went with checking if `://` is present. If yes, schema is parse. If not, http is added as schema.
I don't have an idea how to do it more clever, apparently it is not that easy looking at google results.